modyn/supervisor/internal/evaluation_result_writer/json_result_writer.py

```diff
@@ -17,7 +18,19 @@ def add_evaluation_data(self, dataset_id: str, dataset_size: int, evaluation_dat
             dataset_results["metrics"].append({"name": metric.metric, "result": metric.result})
         self.results["datasets"].append({dataset_id: dataset_results})
 
-    def store_results(self) -> None:
+
+class JsonResultWriter(_JsonResultWriter):
```

Comment:
I don't get the purpose of this yet

Reply:
Previously we wrote all evaluation calls to dedicated files. The default should now be not to create one file per evaluation request but use the central json that is built by the `pipeline_executor`. To keep the old functionality i renamed the old JsonResultWriter to `DedicatedJsonResultWriter` while the new one doesn't create files on its own.